service/controller/v13/cloudconfig/worker_template.go

```diff
@@ -21,6 +21,7 @@ func (c *CloudConfig) NewWorkerTemplate(customObject v1alpha1.AWSConfig, certs l
 			certs:        certs,
 			customObject: customObject,
 		}
+		params.Hyperkube.Kubelet.Docker.CommandExtraArgs = c.k8sKubeletExtraArgs
```

Comment:
This is to use our retagged pause container because we can't pull from GCR. Change is already done for masters.